### DISCUSSION TITLE
你也害怕錯過嗎?

### DISCUSSION THREAD

**Discussion by Hsucuchia** [1101-GS4719-C/Python-Community-GS4719-C#1202] [General]
總是喜歡學習新知識、總是想知道最新消息、害怕錯過任何最新消息、收集很多文章影片，但沒再打開過。看了好多資訊，卻很快就忘了。結果因為盲目接收太多雜亂內容，反而讓自己陷入焦慮跟混亂。因此有以下建議 :

1. 釐清是否符合自己需求 : 了解什麼資訊對自己才是真正有用，根據自己的需求來過濾龐大的資料
2. 勇敢捨棄不需要的資訊 : 人的時間、精力有限，不可能學完所有東西，可以先捨棄暫時用不到的資訊
3. 留時間給自己消化整理 : 每當看完內容後，要留時間給大腦處理，才能確實將內容轉化成幼校的資訊(ex: 在看完設計技巧的分享文章後，思考可以如何應用或連結其他經驗，讓內容有效地吸收運用。)
4. 被動接收=>主動尋找 : 我們學習跟不上資訊更新的速度，但可以從被動接收所有內容，轉為主動尋找自己需要的資訊

找到適合自己的步調，才不會陷入焦慮，學習也會更有效!!

**Comment by DuxtinTai** [comment]
非常贊同您的建議。學習新知識和掌握最新資訊確實非常重要，但是我們需要知道如何有效地管理這些資訊，以免過度負荷自己的大腦和情緒。
對於害怕錯過的問題，我也深受其苦，對此您提到了一些非常實用的建議，例如釐清自己的需求、勇敢捨棄不需要的資訊、留時間消化整理。這些方法可以幫助我們更有效地處理資訊並更好地應用所學。

**Comment by GuanYann** [comment]
在這個快速發展的資訊時代，我們必須要懂得如何過濾並有效地學習資訊，才能跟上時代的步伐。同時，我也認為要在學習的過程中，保持自己的熱情與興趣，才能夠真正地吸收並運用學到的知識。此外，要注意不要過度依賴資訊科技，也要給自己適當的休息時間和思考空間，才能保持身心健康。

**Comment by zilinng** [comment]
我們現在每天可以接收到的資訊量真的太多太多了，我非常同意作者的觀點，學會篩選哪些資訊是有用的哪些可以忽略是現代每個人都應該要學會的一件事，不然什麼都想要最後只會什麼都沒有，學習的時候野生，雖然我們手邊有很多資料，但也應該由淺入深慢慢消化，一下就想全部學會應該只有少數人能達成。

**Reply by ALAN30359704** [reply]
非常棒的建議！釐清自己的需求和目標是非常重要的，這樣才能更有效地尋找和利用資訊。此外，不僅是捨棄不需要的資訊，還可以學習選擇優質的資訊來源，這樣可以減少學習時的混亂感和焦慮感。

而留下時間消化整理是很重要的，因為記憶和學習都需要時間和反覆。只是閱讀和收集資訊並不足以讓我們真正學習和成長，需要進一步思考和應用，讓資訊轉化成知識和技能。

最後，找到適合自己的學習步調和方式，是非常重要的。每個人的學習風格和節奏都不同，所以找到適合自己的方式可以讓學習更加輕鬆和有效。謝謝您的分享！

**Comment by Jack-1D** [comment]
雖然很贊同這篇文章，但是當我看到身旁的人都在花時間學習酷酷的東西的時候，我總會覺得很焦慮，害怕自己跟不上其他人，結果就導致我好像每個都略懂，但都不會學到很精。我認為先例定一個明確的目標，像是希望自己未來變成什麼樣，來決定現在自己應該學些什麼。

**Comment by shuxian12** [comment]
我自己非常喜歡學習新知識，所以覺得找到適合自己的步調很重要。但我每次想學新事物都時常半途而廢，所以非常能體會這個問題。雖然說每次都有釐清自己的需求，了解哪些資訊對自己才是真正有用的，對於節省時間和精力有很大幫助，留時間給自己消化整理也很重要，確保學習內容被轉化成實際能夠應用的知識。但同時也會經常因為覺得太麻煩而不想去做，這真的非常難處理。

**Comment by orianna901001** [comment]
我蠻認同上面的回覆，想要學習甚麼知識的話，我自己會從生活中遇到的問題著手，去查詢一些酷知識，不然一開始遇到太難的問題就想放棄了。

**Comment by llh26** [comment]
我完全被說中⋯，常常害怕漏掉資訊，所以就把他們都放在safari裡面，甚至現在還有一篇文章沒有看完，但也不知道自己什麼時候會看，就這樣一直被放在裡面，清東西的時候也會跳過它想著：這個還沒看完，不能刪。看完文章作者的分享後，我會試著去嘗試捨棄一些自己目前不需要的資訊，讓自己有更多時間去消化整理。

**Comment by PaulSu0905** [comment]
我的方法是創建一個有效的資訊整理系統：將收集的文章、影片和其他資訊整理成可管理的方式。這可以是建立文件夾、使用筆記軟件或使用書籤工具等。這樣你就可以更輕鬆地找到需要的資訊，並避免遺忘。

**Reply by sjh501** [reply]
這感覺是很有用的方法，只是目前還沒有找到自己適合的軟體。

**Comment by larry40305** [comment]
每個人的學習步調和方式都不同。找到適合自己的學習節奏，逐步培養好的學習習慣和技巧，就能更有效地處理資訊，學習到更多有價值的知識。

**Comment by bestbanana295** [comment]
我覺得我在學習資訊的過程當中，有時候會太執著想要弄懂導致進度拖沓不前，或許我該好好 去過濾那些是需要的那些是不用的資訊才是。

**Comment by ThanatosJun** [comment]
我認為樓主的2.3點非常的好，人的時間精力有限，總得做出取捨，喜好也一樣。我們所學習的必須要學會運用，因為實作更能加深記憶與分析脈絡，讓別人的思維轉化為符合自身的裝備，利用並進化自身能力。

**Comment by Alex11392** [comment]
我也有過這樣的經驗。以前國高中時，總是很喜歡在課堂上抄下老師上課提到的所有筆記，不感漏掉一句，生怕漏掉的就在考卷中出現，卻也因此錯失上課專心聽講的機會。

**Comment by lin157644** [comment]
我想我也是相當害怕錯過的人，我的瀏覽器中總是滿滿一排分頁，多到實在不行了還用插件(OneTab)將其存起來，即便我深知我沒有足夠的時間和精力會去看那些存下來的文章網頁，害怕落後同儕的壓力使我不斷努力接收資訊，我想我需要立定好屬於自己的目標，才能審慎分類資訊的必要程度。
